examples/string_in_file

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
```

Comment:
I had to create a bash script that checks if a string is in file because I could't figure out make and the previous make file rule just didn't work anymore.  I really don't know why.